### PR TITLE
Gesture migration

### PR DESCRIPTION
**Pull request by mortoray**
This migrates several of the gesture and behaviors to use the gesture system. It allows these behaviours to overlap and behave together better.

This PR contains:
- [x] Changelog
- [x] Documentation
- [x] Tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Unreleased
 
+## Gesture
+- The experimental `IGesture` interface has changed. 
+	* The `Significance`, `Priority` and `PriotityAdjustment` have been merged into the single `GetPriority` function.
+	* `OnCapture` is changed to `OnCaptureChanged` and provides the previous capture state
```
Comment (kusma) [inline CHANGELOG.md]: Other multi-level lists use two spaces for indent, please use that in the future instead...

File: Source/Fuse.Gestures/Internal/TwoFinger.uno [hunk 2]
```diff
@@ -77,17 +75,16 @@ namespace Fuse.Gestures.Internal
...
 			_point[0].Down = _point[1].Down = -1;
-			Pointer.ReleaseCapture(this);
 			if (_begun)
 			{
-				_node.EndInteraction(this);
 				if (Ended != null)
 					Ended();
 			}
 			_begun = false;
+			_trackingKeyboard = false;
```
Comment (kusma) [inline Source/Fuse.Gestures/Internal/TwoFinger.uno]: Should this be wrapped in a try-finally block, so we know that `_trackingKeyboard` becomes `false` even if an `Ended`-handler throws?

Thinking about it a bit more, events kinda feel a bit crappy for this, as one event-handler throwing is going to ruin the fun for all the others, leaving them in a limbo-state... Why does programming have to be so awful?

File: Source/Fuse.Gestures/Swipe.uno [hunk 3]
```diff
@@ -210,11 +210,11 @@ namespace Fuse.Gestures
 		}
 		
 		internal static Selector GesturePriorityName = "GesturePriority";
-		GesturePriority _gesturePriority = GesturePriority.Lower;
+		GesturePriority _gesturePriority = GesturePriority.Low;
 		/**
 			The priority of the swiping gesture when competing with other gestures.
 			
-			The default is Higher. 
+			The default is Lower.
```
Comment (kusma) [inline Source/Fuse.Gestures/Swipe.uno]: This change of behavior should be mentioned in the changelog.

File: Source/Fuse.Gestures/Internal/TwoFinger.uno (hunk 2 above)
Comment (mortoray) [inline Source/Fuse.Gestures/Internal/TwoFinger.uno]: I'll reorder. In general I want to call  the event handlers last. I don't know if the entire handling is exception safe though, Gesture as a whole might be an issue, let me check a bit there.

Comment (mortoray) [inline Source/Fuse.Gestures/Internal/TwoFinger.uno]: I did something minimal, but looking at the overall code I can't really "fix" it here.

https://github.com/fusetools/fuselibs/issues/4236

**Pull request merged**